gsy_framework/utils.py

```diff
@@ -238,6 +238,11 @@ def datetime_to_string_incl_seconds(date_time: DateTime) -> str:
     return date_time.format(DATE_TIME_FORMAT_SECONDS)
 
 
+def datetime_to_string(datetime_object: DateTime) -> str:
+    """Convert a datetime object into a string with YYYY-MM-DDTHH:mm format."""
```

Comment:
The type hint suggests, that the input type is a pendulum DateTime object. If this is true, lets rename the method to `pendulum_to_string` in order not to get confused by the "normal" DateTime object that has different and less properties.